contrib/move-string-resource.kts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * USAGE:
+ * Cuts string resources with the name arg3 from the module from arg1
+ * and pastes them to the string resources from the module from arg2.
+ *
+ * Example usage:
+ * kotlinc -script move-string-resource.kts ../app ../core history_from_current_book
+ *
+ * This will cut all strings with the name history_from_current_book from
+ * all strings.xml files within directories with a prefix of values*
+ * in ../app/src/main/res/ and paste them into
+ * their corresponding files within ../core/src/main/res/values*.
+ *
+ */
+import java.io.File
+import kotlin.system.exitProcess
+
+
+if (args.size < 3) {
+  print_correct_usage_and_exit()
+}
+
+var path_to_values = "/src/main/res/"
+var english_values_dirname = "values"
+var source = args[0]
+var destination = args[1]
+var key = "name=\"" + args[2] + "\""
+
+var source_dir = File(source + path_to_values)
+var destination_dir = File(destination + path_to_values)
+validate_input()
+
+var found_values: HashMap<String, String> = HashMap<String, String>()
```

Comment:
This sort of public variable is probably to be discouraged.

Prefer pure function, that given an input return an output and chain these together for a more readable program

Something like `destinationDir.pasteStringResources(sourceDir.cutStringResources(key))`, with kotlin extension functions I think would look quite nice.